extensions/amp-subscriptions/0.1/service-adapter.js

```diff
@@ -79,6 +79,15 @@ export class ServiceAdapter {
   getDialog() {
     return this.subscriptionService_.getDialog();
   }
+
+  /**
+   * Returns login platform based on platform selection
+   *
+   * @return {!Promise}
+   */
+  login() {
```

Comment:
Let's rename to represent a bit more precisely what it does. E.g. "selectPlatformForLogin" or some such.

Reply:
Done